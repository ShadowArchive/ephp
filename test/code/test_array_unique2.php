<?php
$list = array(
  '0',
  null,
  0,
  10,
  '10.0',
  '1e1',
  false,
  ''
);
print "------\nDEFAULT\n------\n";
var_dump(array_unique($list));
print "------\nSTRING\n------\n";
var_dump(array_unique($list, SORT_STRING));
print "------\nNUMERIC\n------\n";
var_dump(array_unique($list, SORT_NUMERIC));
print "------\nREGULAR\n------\n";
var_dump(array_unique($list, SORT_REGULAR));

