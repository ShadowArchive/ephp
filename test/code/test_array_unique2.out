------
DEFAULT
------
array(5) {
  [0]=>
  string(1) "0"
  [1]=>
  NULL
  [3]=>
  int(10)
  [4]=>
  string(4) "10.0"
  [5]=>
  string(3) "1e1"
}
------
STRING
------
array(5) {
  [0]=>
  string(1) "0"
  [1]=>
  NULL
  [3]=>
  int(10)
  [4]=>
  string(4) "10.0"
  [5]=>
  string(3) "1e1"
}
------
NUMERIC
------
array(2) {
  [0]=>
  string(1) "0"
  [3]=>
  int(10)
}
------
REGULAR
------
array(4) {
  [0]=>
  string(1) "0"
  [1]=>
  NULL
  [3]=>
  int(10)
  [7]=>
  string(0) ""
}
